packages/wallets/src/components/OptionsAndMultipliersListing/OptionsAndMultipliersListing.tsx

```diff
@@ -129,10 +128,11 @@ const OptionsAndMultipliersListing: React.FC<TOptionsAndMultipliersListingProps>
             >
                 {optionsAndMultipliersContent.map(account => {
                     const title = account.title;
+
                     return (
                         <TradingAccountCard
                             {...account}
-                            key={`trading-account-card-${account.title}`}
+                            key={`trading-account-card-${title}`}
```

Comment:
I know it was already here implemented previously, but looks like potential issue so have to comment - can title be duplicated? 
ideally, we should use something which is unique and will be undoubtedly unique in future - e.g. loginid or something like that.

Reply:
In this case the risk is unlikely, because we are just mapping over a constant 😅 :
```
export const optionsAndMultipliersContent = [
    {
        description: 'Options and multipliers trading platform.',
        icon: <IcAppstoreDerivTrader />,
        redirect: '/',
        smallIcon: <IcAppstoreDerivTrader height='3.2rem' width='3.2rem' />,
        title: 'Deriv Trader',
    },
    {
        description: 'Automate your trading, no coding needed.',
        icon: <IcAppstoreDerivBot />,
        redirect: '/bot',
        smallIcon: <IcAppstoreDerivBot height='3.2rem' width='3.2rem' />,
        title: 'Deriv Bot',
    },
    {
        description: 'Our legacy options trading platform.',
        icon: <IcAppstoreSmartTrader />,
        isExternal: true,
        redirect: getUrlSmartTrader(),
        smallIcon: <IcAppstoreSmartTrader height='3.2rem' width='3.2rem' />,
        title: 'SmartTrader',
    },
    {
        description: 'Our legacy automated trading platform.',
        icon: <IcAppstoreBinaryBot />,
        isExternal: true,
        redirect: getUrlBinaryBot(),
        smallIcon: <IcAppstoreBinaryBot height='3.2rem' width='3.2rem' />,
        title: 'Binary Bot',
    },
    {
        description: 'Trade on the go with our mobile app.',
        icon: <IcAppstoreDerivGo />,
        isExternal: true,
        redirect: getStaticUrl('/deriv-go'),
        smallIcon: <IcAppstoreDerivGo height='3.2rem' width='3.2rem' />,
        title: 'Deriv GO',
    },
];
```